### PR TITLE
Bug 2067820: Update prometheus client_golang from 1.11.0 => 1.11.1

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,4 +19,7 @@ require (
 	k8s.io/klog/v2 v2.30.0
 )
 
-replace vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
+replace (
+	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1 // CVE-2022-21698
```
Comment (adambkaplan) [inline go.mod:24]: We have an explicit dependency on client_golang above, please change it there.

Comment (coreydaley) [reply go.mod]: Updated, ptal

**Review by adambkaplan — APPROVED** [review]
/approve

/lgtm

**Pull request merged**